addon/components/time-series-chart.js

```diff
@@ -361,18 +403,30 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
 
   // If there is a dynamic x axis, then assume the value that it is given,
   // and if it is not a dynamic x axis, set it to the number of x axis ticks.
+  //
   // For a dynamic x axis, let the max number of labels be the minimum of
   // the number of x ticks and the assigned value. This is to prevent
   // the assigned value from being so large that labels flood the x axis.
-  maxNumberOfLabels: Ember.computed('numXTicks', 'dynamicXAxis', function(key, value){
+  maxNumberOfLabels: Ember.computed('numXTicks', 'dynamicXAxis', 'maxNumberOfRotatedLabels', function(key, value){
+    var allowableTicks = this.get('xAxisVertLabels') ? this.get('maxNumberOfRotatedLabels') : this.get('numXTicks');
+
```

Comment:
- We need to add this.xAxisVertLabels as a CP dependency for this property.
- This property can't be set anyway (the object referenced by `value` is ignored), so I would change the pre-existing `function(key, value)` to be just `function()`.